### ISSUE TITLE
[Accessibility] "Go to Recent Directory (Control+G)" instruction is not included in terminal a11y help when cmd prompt is used

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#175175]
CC @meganrogge

The terminal help instruction on "Go to Recent Directory (Control+G)" is only available when integrated shell is used. Try Alt+F1 in cmd prompt. You won't see that.

**Comment by meganrogge** [comment]
Thank you for the issue and clarification.

That is by design because the command isn't supported/doesn't exist without shell integration. This is because we can only have knowledge of cwds via shell integration insights

**Comment by jooyoungseo** [comment]
@meganrogge Ctrl+R does not work though.

**Comment by meganrogge** [comment]
You are correct 👍🏼

**Comment by meganrogge** [comment]
To verify, disable shell integration or use a shell without shell integration support like command prompt and check that the terminal accessibility help menu includes info on the go to recent directory command
